qpup_hw/src/qpup_hw_real.cpp

```diff
@@ -210,9 +211,79 @@ void QPUPHWReal::write(const ros::Time & /*time*/, const ros::Duration & /*perio
       }
     }
 
-    bool successful_joint_write = false;
     uint8_t outgoing_can_data_buffer[CAN_MAX_DLEN];
 
+    // Controller Mode Command (Control Mode + Input Mode)
+    static uint8_t last_control_mode = 0;
```

Comment:
Static isn't great since then you can't have more than one qpup_hw_real. But it's fine for this project lol